Example/Core/Tests/FIROptionsTest.m

```diff
@@ -81,13 +81,14 @@ - (void)testDefaultOptions {
 
 - (void)testInitCustomizedOptions {
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
+#pragma clang diagnostic ignored "-Wnonnull"
```

Comment:
Instead of using `pragma` to ignore nullability warnings, you can also tell the compiler your `nil` is actually nonnull. :D 

`(id _Nonnull)nil`